src/components/DisplayNames/DisplayNamesWithTooltip.tsx

```diff
@@ -61,16 +62,15 @@ function DisplayNamesWithToolTip(props) {
                               displayName={displayName}
                               login={login}
                               avatar={avatar}
-                              textStyles={props.textStyles}
+                              textStyles={textStyles}
                               childRefs={childRefs}
-                              addComma={index < props.displayNamesWithTooltips.length - 1}
```

Comment:
why removing that?

Reply:
@kubabutkiewicz I haven't found it's being used in the component it's passed to